README.md

```diff
@@ -19,6 +19,42 @@
 
 ## Usage
 
+### Clojure CLI Tool
+
+Starting from Clojure CLI v1.11.1.1139,
+[tool](https://clojure.org/reference/deps_and_cli#tool_install)
+installation is supported.
+
+```bash
+# Install tool (latest version)
+clojure -Ttools install-latest :lib com.github.scarletcomply/license-finder :as license-finder
```

Comment:
This can be tested only after the merge (and adding a tag for a new release I guess).
In case you want to try installing and using the tool before the merge, you can install from my fork:

```
clj -Ttools install com.github.pilosus/license-finder '{:git/sha "141fc6f967a6600634fc2f282a8994ab463e347a"}' :as license-finder
```

Alternatively, one can git clone the fork, checkout to the PR branch and add a tool localy via:

```
# ~/.clojure/tools/license-finder.edn

{:lib com.github.pilosus/license-finder,
 :coord
 {
  :local/root "../../path/to/license-finder-fork"
  }}
```